.rubocop.yml

```diff
@@ -6,12 +6,12 @@ require:
 
 AllCops:
   NewCops: enable
-  TargetRubyVersion: 3.0
+  TargetRubyVersion: 3.1
   DisplayStyleGuide: true
   ExtraDetails: true
   Exclude:
-    - 'bin/**/*'
-    - 'vendor/bundle/**/*'
+    - "bin/**/*"
+    - "vendor/bundle/**/*"
```

Comment:
Since we are here, we can also improve this, since adding `Exclude` will overwrite existing settings


The default `Exclude:` is not inherited when a custom value is provided,
so the performance was degraded because time was spent searching in
`.git`.

Ref: rubocop/rubocop#9325

```
    - .git/**/*
    - .github/**/*
    - bin/**/*
    - coverage/**/*
    - doc/**/*
    - vendor/**/*
```

Reply:
I've added this change.